v2/emailpassword/serverless/with-aws-lambda/jwt-authorizer.mdx

```diff
@@ -75,10 +91,81 @@ function getBackendConfig(): SuperTokensTypes.TypeInput {
 
 module.exports.getBackendConfig = getBackendConfig;
 
+```
+</TabItem>
+
+<TabItem value="python">
+
+```python title="config.py"
+from supertokens_python.recipe import session
+from supertokens_python import (
+    InputAppInfo,
+    SupertokensConfig,
+)
+
+from typing import Any, Dict, Optional
+
+supertokens_config = SupertokensConfig(
+    ^{coreInjector_connection_uri_comment_with_hash}
+    connection_uri=^{coreInjector_uri}
+    ^{coreInjector_api_key_commented_with_hash}api_key=^{coreInjector_api_key}
+)
+
+app_info = InputAppInfo(
+    # learn more about this on https://supertokens.com/docs/session/appinfo
+    app_name="^{form_appName}",
+    api_domain="^{form_apiDomain}",
+    website_domain="^{form_websiteDomain}",
+    api_base_path="^{form_apiBasePath}",
+    website_base_path="^{form_websiteBasePath}",
+    api_gateway_path="/dev",
+)
+
+framework = "fastapi"
+
+# highlight-start
+from supertokens_python.recipe.session.interfaces import RecipeInterface as SessionRecipeInterface, SessionContainer
+
+def override_session_functions(oi: SessionRecipeInterface) -> SessionRecipeInterface:
+    oi_create_new_session = oi.create_new_session
+
+    async def create_new_session(
+            request: Any, user_id: str, access_token_payload: Optional[Dict[str, Any]], 
+            session_data: Optional[Dict[str, Any]], user_context: Dict[str, Any]
+        ) -> SessionContainer:
+        # AWS requires JWTs to contain an audience (aud) claim
+        # The value for this claim should be the same as the
+        # value you set when creating the authorizer
+
+        # highlight-next-line
+        if access_token_payload is not None:
```

Comment:
what if access_token_payload is none? Instead, check if it is none and then make it an empty map before adding the "aud" claim to it

Reply:
Done.